content/blog/2021-09-14-september-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+﻿---
+title: September'21 Heartbeat
```

Comment:
minor: should we start naming heartbeats w/o `dvc` as we do for the gems?

Reply:
Sure!  Will correct!

Reply:
@shcheklein  Went to make changes and this happened.  I don't know why these docs are on Sept-Heartbeat

![image](https://user-images.githubusercontent.com/34289199/133344576-b9828c53-84ae-4de2-9d92-6bcd3ecd68b4.png)

Reply:
that's fine ... I suggest we merge it as is (I can fix the link right here from the UI) and we can do the new name later, wdyt?